projects/ngworker/lumberjack/http-driver/package.json

```
@@ -0,0 +1,10 @@
+{
+  "ngPackage": {
+    "lib": {
+      "entryFile": "./src/public_api.ts"
+    }
+  },
+  "peerDependencies": {
+    "@angular/common/http": "^10.0.14"
```

We need to list all dependencies that are used in this sub-module.
```suggestion
    "@angular/common/http": "^10.0.14",
    "@angular/core": "^10.0.14",
    "rxjs": "^6.5.5"
```

---

`@angular/common/http` is not a separate NPM package so it's not a valid dependency to list here. We should list `@angular/common` here, but remove it from our primary entry point and remove it from the `imports` metadata option of `LumberjackModule`. That import doesn't make sense since we don't have any declarations in that Angular module.